backend/daemon/api/entities/v1alpha/entities_test.go

```diff
@@ -170,3 +176,75 @@ func TestEntityTimeline(t *testing.T) {
 
 	testutil.ProtoEqual(t, want, timeline, "timeline must match")
 }
+
+func TestDeleteEntity(t *testing.T) {
+	//TODO(juligasa): this unit test is not really unitary since it depends on documents API.
+	db := storage.MakeTestDB(t)
+
+	docsapi := newTestDocsAPI(t, db, "alice")
+	ctx := context.Background()
+
+	blobs := hyper.NewStorage(db, zap.NewNop())
```

Comment:
Looks like a separate storage was already created inside newTestDocsAPI. Not sure if this could cause any issues, but maybe it would be better to reuse the same instance.